t/misc_needles/check-workround-hash-bsc7654321-20190522.json

```diff
@@ -0,0 +1,19 @@
+{
+    "area": [
+        {
+            "height": 100,
+            "match": 80,
+            "type": "match",
+            "width": 200,
+            "xpos": 10,
+            "ypos": 10
+        }
+    ],
+    "tags": ["check-workaround-hash"],
+    "properties": [
```

Comment:
Seems like none of these JSON files contain multiple properties. Maybe you can add some other text-only and HASH-properties before and after the workaround property. It shouldn't prevent the workaround property to be found and it would make sense to test that.

Reply:
added